test/system/060-mount.bats

```diff
@@ -308,6 +308,11 @@ EOF
 }
 
 @test "podman mount noswap memory mounts" {
+    kern=$(uname -r | awk -F. '{print $1}')
+    if [[ $kern -lt 6 ]]; then
+        skip "tmpfs + noswap requires kernel >= 6.something"
+    fi
```

Comment:
version checks are bad, especially in a RHEL world where such features may be backported at any time
can you do a check like `mount -t tmpfs -o noswap  none /tmp/....` and if this works we have support for it. (don't forget to umount in this case)

Reply:
Thanks, I'll try. Am also working on a fix for the multi-pod test, so this is now draft.

Reply:
Also if you do such a check remember that this is only supported as root, it is not allowed as rootless.